src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -214,6 +214,17 @@ protected ResourceLocation getModelLocation(ResourceLocation model)
     {
         return new ResourceLocation(model.getResourceDomain(), model.getResourcePath() + ".json");
     }
+    
+    @Override
+    public ModelBlockDefinition getModelBlockDefinition(ResourceLocation location)
+    {
+        return super.getModelBlockDefinition(location);
+    }
+    
+    public ResourceLocation getBlockStateLocation(ResourceLocation location)
```

Comment:
This was added as a means to easily get the resource location that points to the blockstate folder.